actors/common/src/main/java/org/sunbird/common/ShadowUserProcessor.java

```diff
@@ -153,6 +157,23 @@ private void updateUser(ShadowUser shadowUser) {
 
     }
 
+
+    private void generateTelemetry(String userId,String rootOrgId,ShadowUser shadowUser){
+        ExecutionContext.getCurrent().setRequestContext(getTelemetryContextByProcessId((String) shadowUser.getProcessId()));
+        ProjectLogger.log("ShadowUserProcessor:generateTelemetry:generate telemetry:" + shadowUser.toString(), LoggerEnum.INFO.name());
+        Map<String, Object> targetObject = new HashMap<>();
+        Map<String, String> rollUp = new HashMap<>();
+        rollUp.put("l1", rootOrgId);
+        List<Map<String, Object>> correlatedObject = new ArrayList<>();
+        ExecutionContext.getCurrent().getRequestContext().put(JsonKey.ROLLUP, rollUp);
+        TelemetryUtil.generateCorrelatedObject(shadowUser.getProcessId(), JsonKey.PROCESS_ID, null, correlatedObject);
+        targetObject =
```

Comment:
Since job is done by scheduler , so actor will be either System or job and it will have either job id or uuid.